src/utils.ts

```diff
@@ -35,5 +35,34 @@ export function getRequiredRosDistributions(): string[] {
 			RegExp("\\s")
 		);
 	}
+
+	if (!validateDistro(requiredRosDistributionsList)) {
+		throw new Error("Input has invalid distribution names.");
+	}
+
 	return requiredRosDistributionsList;
 }
+
+//list of valid linux distributions
+const validDistro: string[] = [
+	"kinetic",
+	"lunar",
+	"melodic",
+	"noetic",
+	"dashing",
+	"eloquent",
+	"foxy",
+];
+
+//go through requiredRosDistributionsList and validate every linux distribution input
+function validateDistro(requiredRosDistributionsList: string[]): boolean {
+	for (let rosDistro of requiredRosDistributionsList) {
+		//If the validDistro string array doesn't contain the existing input
+		//Return false
```

Comment:
I think that these 2 lines of comment are rot-prone because they add no context and just describe the code, verbatim. I'd remove them